File: prak3_java/Disk.java
public class Disk {
    
    private String type;
    private String capacity;
    private int price;

    //konstruktor
    public Disk() {
    }
    
    public Disk(String type, String capacity, int price) {
        this.type = type;
        this.capacity = capacity;
        this.price = price;
    }

    //getter setter
    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getCapacity() {
        return capacity;
    }

    public void setCapacity(String capacity) {
        this.capacity = capacity;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

}

File: prak3_c++/Disk.cpp
#include <string>

using std::string;

class Disk {
    private:
        string type;
        string capacity;
        int price;

    public:
        //konstruktor
        Disk(){
        }

        Disk(string t, string c, int p){
            type = t;
            capacity = c;
            price = p;
        }


        //setter
        void setType(string t){
            type = t;
        }

        void setCapacity(string c){
            capacity = c;
        }

        void setPrice(int p){
            price = p;
        }

        //getter
        string getType(){
            return type;
        }

        string getCapacity(){
            return capacity;
        }

        int getPrice(){
            return price;
        }
        

        //destruktor
        ~Disk(){
        }

};
